### PR TITLE
Column description for Default Pages configuration

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds  column description for Default Pages configuration

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/cms/pages-default.html

### CODE REVIEW

File: src/cms/pages-default.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ The _Default Pages_ configuration determines the landing page that is associated
    ![]({% link images/images/config-general-web-default-pages.png %}){: .zoom}
    [_Default pages_]({% link configuration/general/web.md %})
 
+   {: .field-table }
+   |Field|[Scope]({% link configuration/scope.md %})|Description|
+   |--- |--- |--- |
+   |Default Web URL|Store View|Indicates the landing page that is associated with the base URL. This is set by default to “cms” to indicate a page from Magento’s content management system. You can also use a different type of landing page, such as a blog. For example, if a blog is installed on the server at magento/blog, you can enter the name of the “blog” folder as a relative path to the selection of pages.|
```
Comment (leslietilling) [inline src/cms/pages-default.md:21]:
```suggestion
   |Default Web URL|Store View|Indicates the landing page that is associated with the base URL. This is set by default to `cms` to indicate a page from the Magento content management system. You can also use a different type of landing page, such as a blog. For example, if a blog is installed on the server at magento/blog, you can enter the folder name `blog` as a relative path to the selection of pages.|
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**